aichatweb-aspire.AppHost/Program.cs

```diff
@@ -0,0 +1,30 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var qdrantApiKey = builder.AddParameter("qdrantApiKey", secret: true);
+
+var ollama = builder.AddOllama("ollama")
+    .WithDataVolume();
+var chat = ollama.AddModel("chat", "llama3.2");
+var embeddings = ollama.AddModel("embeddings", "all-minilm");
+
+var vectorDB = builder.AddQdrant("vectordb", apiKey: qdrantApiKey, grpcPort: 6334, httpPort: 6333)
+    .WithDataBindMount("./qdrant_data")
```

Comment:
Why a bind mount instead of a volume here? Bind mounts are typically only used when you want to give the container access to data on the host. If you just want data to be persistent between container instances you're better off using a volume.

```suggestion
    .WithDataVolume()
```

Reply:
Thanks!